Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,515 @@
+
+The non-physical deliverables of this project will be the project culmination report which will include a very detailed examination of the projects
+results. These results include the expected performance versus the test performance, the nozzle dimension polynomial approximation, the structure performance, and the
+propulsion system performance. The structural performance is a section that discusses the reasoning behind the initial design and any problems
+that were found during testing. The problems to be avoided will be, unecessary frictional loss due to bad connection points or instability of the test stand due to
+bad interfacing between structural components. The propulsion system performance will be very similar to this except it will focus on slightly different aspects, such as
+the mass flow under various ambient conditions and temperatures. The propulsion performance will mostly be measured through the data analysis. This effects the
+specific impulse. It will also concern the expansion of the flow, as the maximum thrust will occur when the exit pressure equals the ambient pressure and thus the shocks
+in the flow will expand optimally. This will be observed through schlieren lensing, as it has been performed before during nozzle testing. This was performed in the Spring 2017
+semester for flow visualization. The procedure was; turn the lights off, put the nozzle on the air hose, put a white piece of paper on a table, put it parallel to the table, initiate the cold burn, use a phone light
+to project the flow onto the white paper below it. Then distance between the nozzle and the light and the table were toggled until the best focus point was found. \autoref{fig: nozzle shocks}
+shows the exiting flow from a 3D printed nozzle.
+
+\begin{figure}[h]
+  \includegraphics[width=\linewidth]{figs/nozzle-shocks[2].png}
+  \caption{The flow is seen here exiting a small 3D printed supersonic nozzle. The procedure was not unlike schlieren lensing, however the clarity was much lower.}
+  \label{fig: nozzle-shocks}
+\end{figure}
+
+\subsection{Milestones}
+\label{subsec:milestones}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+
+There is a preliminary plan for the semester that is displayed in further detail in \autoref{table: fall-plan}. The following list will display the description and timing
+of the different phases; design, sanity tests, experimentation, and data analysis.
+
+\subsubsection{Design Phase}
+\label{subsubsec: design phase}
+    The goals are to design the propulsion system, the test stand, the nozzle. At the end of this phase
+    there will be a full system CAD model and a series of CAD nozzles to be fabricated. This CAD model is essential for the next phase, as the fabrication
+    will use it as a guide (inlcuding drawings for machinists).
+
+    \textit{Projected Duration: 2-3 weeks}
```

Comment:
Turns out `-`, `--` and `---` mean different things in LaTeX. `-` is a hyphen, `--` is an *en* dash, and `---` is an *em* dash.

`--` proper in this case when you are referring to a range.
https://tex.stackexchange.com/questions/3819/dashes-vs-vs